src/library/crate/cratefeature.cpp

```diff
@@ -36,6 +37,9 @@ QString formatLabel(
         crateSummary.getTrackDurationText());
 }
 
+const ConfigKey kConfigKeyLastImportExportCrateDirectoryKey(
+        "[Library]", "last_import_export_crate_directory");
+
```

Comment:
@uklotzde I'd like to hear your opinion on this. Instead of defining `kConfigKeyLastImportExportCrateDirectoryKey` in a bunch of cpp files all at once, put it in a header file under the `mixxx` namespace and then `#include` it where its needed. Does that make sense, what do you need to specify for it to work? something like `inline const ConfigKey ...`?